program-runtime/src/invoke_context.rs

```diff
@@ -43,6 +43,7 @@ pub type ProcessInstructionWithContext = fn(&mut InvokeContext) -> Result<(), In
 pub struct BuiltinProgram {
     pub program_id: Pubkey,
     pub process_instruction: ProcessInstructionWithContext,
+    pub default_compute_unit_cost: u64,
```

Comment:
`default_compute_unit_cost` will be used in follow-up PR to consume units from TX's compute budget when builtins are processed.